Classes/Information/EventInformation.php

```diff
@@ -11,15 +11,30 @@
 
 namespace FriendsOfTYPO3\Kickstarter\Information;
 
-readonly class EventInformation
+use FriendsOfTYPO3\Kickstarter\Information\Normalization\EventClassNameNormalizer;
+use FriendsOfTYPO3\Kickstarter\Information\Normalization\UseNormalizer;
+use FriendsOfTYPO3\Kickstarter\Information\Validatation\EventClassValidator;
+use FriendsOfTYPO3\Kickstarter\Information\Validatation\UseValidator;
+
+class EventInformation implements InformationInterface
 {
     private const EVENT_PATH = 'Classes/Event/';
 
+    private ?ExtensionInformation $extensionInformation = null;
+    #[UseValidator(EventClassValidator::class)]
```

Comment:
Is this the right position? Property? Shouldn't it be a class attribute?

Reply:
Nope we are validating each property singulary (here that class name must end on Event and must be CamelCase etc). Later on we could also introduce DTO wide validators